doc/network_chaos.md

```diff
@@ -0,0 +1,107 @@
+# Network Chaos Document
+
+This document whill help user to build Network Chaos experiments.
+
+Network chaos are mainly divided into two categories, namely **netem chaos** and **network partition**.
+
+Netem chaos contains some kinds of network chaos, such as delay, duplication, loss and corrupt.
+
+Network partition can decompose pods into several independent subnets by blocking communication between them.
+
+
+# Network Partition Action
+Sample network partition ducument:
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: network-partition-example
+  namespace: chaos-testing
+spec:
+  action: partition
+  mode: one
+  selector:
+    namespaces:
+      - tidb-cluster-demo
+    labelSelectors:
+      "app.kubernetes.io/component": "tikv"
+  direction: to
+  target:
+    selector:
+      namespaces:
+        - tidb-cluster-demo
+      labelSelectors:
+        "app.kubernetes.io/component": "tikv"
+    mode: one
+  duration: "10s"
+  scheduler:
+    cron: "@every 15s"
+```
+User can find and edit the template refer to [examples/network-partition-example.yaml](../examples/network-partition-example.yaml).
+* **action** defines the specific pod chaos action. In this case, it means network partition, represents the chaos action of network partition of pods.
+* **mode** defines the mode to run chaos action.
+* **selector** is used to select pods that are used to inject chaos action.
+* **direction** represents the partition direction, supported direction: from / to / both.
+* **target** represents network partition target.
+* **duration** define the duration time for each chaos experiment. As the example shows, the network partition lasts 10 seconds.
+* **scheduler** defines some scheduler rules to the running time of the chaos experiment about pods. More rule info: https://godoc.org/github.com/robfig/cron
+
+
+# Netem Chaos Actions
```

Comment:
```suggestion
# Netem Chaos Actions
## Netem Chaos Actions
```

Reply:
Directory at this level has been modified.